### PR TITLE
Added DBA.FR75 and related conformance test.

### PR DESCRIPTION
**Pull request by allenbross-wf**
#### Reason for change

> DBA.FR75: The company must provide information on the number of employees. The rule is activated if personnel costs (fsa:EmployeeBenefitsExpense) or salaries (fsa:WagesAndSalaries) are greater than DKK 200,000, and no number of employees (fsa:AverageNumberOfEmployees) has been specified.

#### Description of change

Added DBA.FR75 and related testcase.

#### Steps to Test

CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: arelle/plugin/validate/DBA/ValidationPluginExtension.py [hunk 1]
```diff
@@ -11,12 +11,15 @@
...
 
+DANISH_CURRENCY = 'DKK'
+EMPLOYEE_COST_THRESHOLD = 200000
 NAMESPACE_CMN = 'http://xbrl.dcca.dk/cmn'
 NAMESPACE_FSA = 'http://xbrl.dcca.dk/fsa'
 NAMESPACE_GSD = 'http://xbrl.dcca.dk/gsd'
 NAMESPACE_SOB = 'http://xbrl.dcca.dk/sob'
 
 
+
```
Comment (aaroncameron-wk) [inline arelle/plugin/validate/DBA/ValidationPluginExtension.py:22]: Extra whitespace

File: arelle/plugin/validate/DBA/rules/fr.py [hunk 2]
```diff
@@ -456,6 +457,42 @@ def rule_fr56(
...
+    """
+    costModelFacts = []
+    employeeBenefitsExpenses = val.modelXbrl.factsByQname.get(pluginData.employeeBenefitsExpenseQn, set())
+    wagesAndSalaries = val.modelXbrl.factsByQname.get(pluginData.wagesAndSalariesQn, set())
+    employeeFacts =employeeBenefitsExpenses.union(wagesAndSalaries)
+    for fact in employeeFacts:
+        if int(fact.value) > EMPLOYEE_COST_THRESHOLD and fact.unit.value == DANISH_CURRENCY:
+            costModelFacts.append(fact)
+    if costModelFacts:
+        return errorOnRequiredFact(
```
Comment (aaroncameron-wk) [inline arelle/plugin/validate/DBA/rules/fr.py:482]: I think you want `yield` here, and the `else` block below isn't necessary.

File: arelle/plugin/validate/DBA/rules/__init__.py [hunk 3]
```diff
@@ -58,20 +58,32 @@ def errorOnRequiredFact(
...
     facts: set[ModelFact] = modelXbrl.factsByQname.get(factQn, set())
     if facts:
         for fact in facts:
             if not fact.isNil:
                 return
-    yield Validation.error(
-        codes=code,
-        msg=message,
-)
+    if level == 'error':
```
Comment (aaroncameron-wk) [inline arelle/plugin/validate/DBA/rules/__init__.py:75]: I suggest at least replacing `level: str = 'error'` with `isError: bool = True` so we aren't depending on "magic" strings. But a step further, I might suggest just repurposing this to simply `isFactMissing` and let the caller decide how it wants to call `Validation.error` or `Validation.warning`.

File: arelle/plugin/validate/DBA/rules/fr.py [hunk 4]
```diff
@@ -456,6 +457,37 @@ def rule_fr56(
...
+    than DKK 200,000, and no number of employees (fsa:AverageNumberOfEmployees) has been specified.
+    """
+    costModelFacts = []
+    employeeBenefitsExpenses = val.modelXbrl.factsByQname.get(pluginData.employeeBenefitsExpenseQn, set())
+    wagesAndSalaries = val.modelXbrl.factsByQname.get(pluginData.wagesAndSalariesQn, set())
+    employeeFacts =employeeBenefitsExpenses.union(wagesAndSalaries)
+    for fact in employeeFacts:
+        if int(fact.value) > EMPLOYEE_COST_THRESHOLD and fact.unit.value == DANISH_CURRENCY:
+            costModelFacts.append(fact)
+    if costModelFacts and isFactMissing(val.modelXbrl, pluginData.averageNumberOfEmployeesQn):
```
Comment (aaroncameron-wk) [inline arelle/plugin/validate/DBA/rules/fr.py:481]: Fine to merge, but we can skip a lot of work in this validation if we just check `if not isFactMissing(val.modelXbrl, pluginData.averageNumberOfEmployeesQn)` and exit first thing before we build/check `costModelFacts`.

**Review by aaroncameron-wk — DISMISSED** [review]

**Review by aaroncameron-wk — DISMISSED** [review]

**Review by derekgengenbacher-wf — APPROVED** [review]

**Review by derekgengenbacher-wf — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
QA +10 Testing locally succeeded.

**Pull request merged**